src/js/call.js

```diff
@@ -206,7 +220,14 @@ Call.fetchTurnConfig_ = function(onSuccess, onError) {
     }
 
     var response = JSON.parse(xhr.responseText);
-    onSuccess(response);
+    Call.cachedIceServers_ = response;
+    Call.getCachedIceCredentials_ = function() {
+      // Make a new object due to tests modifying the original response object.
+      return JSON.parse(JSON.stringify(Call.cachedIceServers_));
+    };
+    Call.fetchTurnTime_ = Date.now();
```

Comment:
Better yet... compute here the expireTime and call it:
"Call.cachedTurnConfigExpireTime_"